pkg/cadvisor/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM linuxkit/alpine:07f7d136e427dc68154cd5edbb2b9576f9ac5213 as build
+
+RUN apk add --no-cache bash go git g++ linux-headers make
```

Comment:
are `g++` and `linux-headers` needed? They look like they were copy/pasted from somewhere...

Reply:
Yes they are required, or at least musl-dev and linux-headers. It looks for netlink, pthreads etc.